src/parser/openfga.ne

```diff
@@ -165,7 +165,7 @@ define_but_not  -> define_base _spacing _but_not _spacing define_base {%
 %}
 
 define_but_not_11  -> define_base_11 _spacing _but_not _spacing define_base_11 {%
-    data => ({ base: data[0], diff: data[4], type: "exclusion", allowedTypes: data[0].allowedTypes })
+    data => ({ base: data[0], diff: data[4], type: "exclusion", allowedTypes: data[0].allowedTypes.concat(...data[4].allowedTypes) })
```

Comment:
Anything in the `subtract` portion of the but not (the portion that comes after) should not include directly assignable - we should add validation for that, but can be in a subsequent PR.